virtme/commands/run.py

```diff
@@ -1397,12 +1428,14 @@ def do_it() -> int:
     if args.graphics is None and not args.script_sh and not args.script_exec:
         qemuargs.extend(["-echr", "1"])
 
-        # Redirect kernel errors to stderr, creating a separate console.
-        #
-        # If we don't have access to stderr via procfs (for example when
-        # running inside a container), print a warning and implicitly
-        # suppress the kernel errors redirection.
-        if can_access_file("/proc/self/fd/2"):
+        if args.systemd:
+            pass
```

Comment:
Can you please add a comment why nothing needs to be done here?

Reply:
Added in https://github.com/arighi/virtme-ng/pull/280/commits/4ffb83c45a2782f631b7ec48bfa07edca902ec4c